src/Aspire.Hosting/api/Aspire.Hosting.cs

```diff
@@ -325,6 +382,15 @@ public ExternalServiceResource(string name, System.Uri uri) : base(default!) { }
         public ApplicationModel.ParameterResource? UrlParameter { get { throw null; } }
     }
 
+    public partial interface IDeveloperCertificateService
```

Comment:
@danegsta - are we happy with this API being "stable"? Or do we think it should be experimental to start?